showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -186,19 +189,34 @@ private fun ShowkaseAppBarTitle(
     }
 }
 
+val LineCountKey = SemanticsPropertyKey<Int>("lineCount")
```

Comment:
Should this be in its own file? :) What do u think?

Reply:
Maybe move to a file called `SemanticsUtils` or something along those lines

Reply:
Good idea! Moved it into `SemanticsUtils` in [833e234](https://github.com/airbnb/Showkase/pull/240/commits/833e234c90e82b0070ac40dbab265f4312918787) 😄